bagelbids/models.py

```diff
@@ -1,13 +1,13 @@
 import uuid
 from typing import List, Literal, Optional, Union
 
-from pydantic import BaseModel, Field, HttpUrl
+from pydantic import BaseModel, Extra, Field, HttpUrl
 
 UUID_PATTERN = r"[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$"
 BAGEL_UUID_PATTERN = r"^bg:" + UUID_PATTERN
 
 
-class Bagel(BaseModel):
+class Bagel(BaseModel, extra=Extra.forbid):
```

Comment:
ah cool, this is how we tell pydantic to not allow any additional properties, yes?

Reply:
yes! and from my understanding/testing this config option is inherited by any class that inherits from `Bagel` :)

Reply:
that makes sense! Nice.